mappings/platform_file_centric.json

```diff
@@ -0,0 +1,318 @@
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "primary_site": {
+        "type": "keyword"
+      },
+      "object_id": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_type": {
+        "type": "keyword"
+      },
+      "data_type": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_access": {
+        "type": "keyword"
+      },
+      "analysis": {
+        "properties": {
+          "analysis_id": {
+            "type": "keyword",
+            "copy_to": "file_autocomplete"
+          },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "experiment": {
+            "type": "object",
+            "properties": {
+              "platform": {
+                "type": "keyword"
+              },
+              "platform_model": {
+                "type": "keyword"
+              },
+              "sequencing_date": {
+                "type": "keyword"
+              },
+              "library_name": {
+                "type": "keyword"
+              },
+              "library_strategy": {
+                "type": "keyword"
+              },
+              "sequencing_center": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      "dynamic": {
+        "properties": {
+          "workflow_name": {
+            "type": "keyword"
+          },
+          "workflow_type": {
+            "type": "keyword"
+          },
+          "analysis_software": {
+            "type": "keyword"
+          }
+        }
+      },
+      "file": {
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          },
+          "md5sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          }
+        }
+      },
+      "index_file": {
+        "properties": {
+          "object_id": {
+            "type": "keyword"
+          },
+          "format": {
+            "type": "keyword"
+          },
+          "md5_sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          }
+        }
+      },
+      "donors": {
+        "type": "nested",
+        "properties": {
+          "id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "submitter_donor_id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "gender": {
+            "type": "keyword"
+          },
+          "vital_status": {
+            "type": "keyword"
+          },
+          "survival_time": {
+            "type": "integer"
```

Comment:
`integer` type?

Reply:
@junjun-zhang i dont understand what is wrong here - why should survival time not be an integer?

Reply:
sorry, I bad, I thought it was `keyword`.